HSLdevcom/digitransit-ui
# DT-3460: Empty input field if use current location text is in it

## Proposed Changes

Steps how to test: 
  - Go to mobile view in Ui.
  - From input field select Select from map option
  - Choose a location that doesn't have an address (Popup says Current Location, or something similar)
  - Click the input field again
 - Text Selected Location should be gone as requested.


## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request
